lib/puppet/provider/vcenter.rb

```diff
@@ -1,28 +1,12 @@
 # Copyright (C) 2013 VMware, Inc.
-begin
-  require 'puppet_x/puppetlabs/transport'
-rescue LoadError => e
-  require 'pathname' # WORK_AROUND #14073 and #7788
-  vmware_module = Puppet::Module.find('vmware_lib', Puppet[:environment].to_s)
-  require File.join vmware_module.path, 'lib/puppet_x/puppetlabs/transport'
-end
+#
 
-begin
-  require 'puppet_x/puppetlabs/transport/vsphere'
-rescue LoadError => e 
-  require 'pathname' # WORK_AROUND #14073 and #7788
-  module_lib = Pathname.new(__FILE__).parent.parent.parent
-  require File.join module_lib, 'puppet_x/puppetlabs/transport/vsphere'
-end
+module_lib = Pathname.new(__FILE__).parent.parent.parent
+
+require File.join module_lib, 'puppet_x/vmware/vmware_lib/puppet_x/puppetlabs/transport'
+require File.join module_lib, 'puppet_x/vmware/vmware_lib/puppet_x/puppetlabs/transport/vsphere'
```

Comment:
@maniacmurphy This PR is removing the rescue block and is already requiring transport/vsphere from itself, as you have described....